src/isar_exr/api/energy_robotics_api.py

```diff
@@ -205,19 +207,22 @@ def wake_up_robot(self, exr_robot_id: str) -> None:
         except Exception as e:
             raise RobotException(e)
 
-        startTime = datetime.now()
-        while not self.is_robot_awake(exr_robot_id):
-            if (datetime.now() - startTime).seconds > settings.MAX_WAKE_TIME_S:
+        awake = self.is_robot_awake(exr_robot_id)
+        while not awake:
+            time_passed_since_function_call = (
+                datetime.today() - startTime
+            ).total_seconds()
+            if time_passed_since_function_call > settings.MAX_TIME_FOR_WAKEUP:
                 raise RobotException(
-                    f"Not able to wake up robot after '{settings.MAX_WAKE_TIME_S}' seconds."
+                    f"Not able to wake up robot after '{settings.MAX_TIME_FOR_WAKEUP}' seconds."
```

Comment:
I think this won't work? `awake` is never updated inside the while loop so the while loop will always time out
The previous method of while not self.is_robot_awake works though